plugins/ufc-scoreboard/scroll_display.py

```
@@ -0,0 +1,377 @@
+            try:
+                ufc_icon = Image.open(self.UFC_SEPARATOR_ICON)
+                if ufc_icon.mode != "RGBA":
+                    ufc_icon = ufc_icon.convert("RGBA")
+                aspect = ufc_icon.width / ufc_icon.height
+                new_width = int(separator_height * aspect)
+                ufc_icon = ufc_icon.resize(
+                    (new_width, separator_height), LANCZOS
+                )
+                self._separator_icons["ufc"] = ufc_icon
+                self.logger.debug(f"Loaded UFC separator icon: {new_width}x{separator_height}")
+            except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)